app/src/main/cpp/skyline/input/touch.cpp

```diff
@@ -15,28 +15,34 @@ namespace skyline::input {
         }
     }
 
-    void TouchManager::SetState(const span<TouchScreenPoint> &points) {
+    void TouchManager::SetState(const span<TouchScreenPoint> touchPoints) {
         if (!activated)
             return;
 
         const auto &lastEntry{section.entries[section.header.currentEntry]};
         auto entryIndex{(section.header.currentEntry != constant::HidEntryCount - 1) ? section.header.currentEntry + 1 : 0};
         auto &entry{section.entries[entryIndex]};
+        auto points{touchPoints.subspan(0, std::min(touchPoints.size(), entry.data.size()))};
```

Comment:
```suggestion
        touchPoints = touchPoints.first(std::min(touchPoints.size(), entry.data.size()));
```

Reply:
You've stolen my amazing CR comment, I can't believe this.